Controllers/LeaderboardController.cs

```diff
@@ -300,6 +301,75 @@ public class LeaderboardController : Controller {
                 foreach (var score in leaderboard.Scores) {
                     score.Player = PostProcessSettings(score.Player);
                 }
+
+                // TODO: IS THIS THE RIGHT WAY TO DO THIS?
```

Comment:
Yeah, but you ignored the `clanRanking` bool from query

Reply:
Fixed; I haven't looked at the website PR yet, so there might be more changes with this, but basically, when clanRanking is false, it means you are just on the 'Players' page of the leaderboard, so we don't need to pull as much data. We still need to pull some clanRanking data because the leaderboardHeader on the website displays who owns the clan/whether the leaderboard is contested/etc. But when its true, pull the associated scores and ALL the clanRankings.

Reply:
Could also not return regular scores if the bool is set. Maybe turn it into an enum?

Reply:
I tried this out, but for some reason the website was crashing in Leaderboard.svelte in createLeaderboardStore if the "Score" was null. It was deep in the weeds of the website and I wasn't able to come up a with a fix. I will try this one more time and if it crashes again, I will ask for some assistance.

Reply:
Fixed